scai-gen/go.mod

```diff
@@ -1,9 +1,9 @@
-module scai-gen
+module github.com/in-toto/scai-demos/scai-gen
 
 go 1.20
 
 require (
-	github.com/adityasaky/in-toto-attestation-verifier v0.0.0-20230821185233-3e31892810a9
+	github.com/in-toto/attestation-verifier
```

Comment:
Was this manually entered? I'm surprised _some_ version information isn't expected here.

Reply:
Yes I entered this manually, now that the repo is transferred over, I'll run `go mod tidy`